app/models/ability.rb

```diff
@@ -24,7 +24,7 @@ def image_editor_permissions
     can [:file_manager, :save_structure], MultiVolumeWork
     can [:create, :read, :edit, :update, :publish, :download], FileSet
     can [:create, :read, :edit, :update, :publish], Collection
-
+    can [:manage], ScannedResource
```

Comment:
Accidental addition. I'll remove it. I was poking at the abilities because of some permissions issues in the tests.